docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -0,0 +1,419 @@
+---
+meta:
+  title: "Next.js Starter Kit"
+  parentTitle: "Guides"
+  description: "Get started with the Liveblocks Next.js Starter Kit"
+---
+
+## Features
+
+- Documents dashboard with pagination, drafts, groups, auto-revalidation
+- Collaborative whiteboard app with fully-featured share menu
+- Access can be granted to users, groups, and the public
+- Authentication compatible with GitHub, Google, Auth0, and more
+- Type-safe client & server functions, and everything documented
+- A great starting point for your collaborative application
+
+## Liveblocks
+
+This starter kit uses [Liveblocks](https://liveblocks.io]) to power its
+collaboration. Here's how to get it running.
+
+1. Create an account at [liveblocks.io](https://liveblocks.io])
+2. Navigate to the [dashboard](https://liveblocks.io/dashboard) and click "Add
+   project..."
+3. Create a new project with "Development" as the environment.
+4. Go to "API keys" on the side menu, reveal your secret key, and copy it
+5. Create a file called `.env.local` alongside `package.json` in your project.
```

Comment:
```suggestion
5. Go to your downloaded starter kit, and enter the file called `.env.local` alongside `package.json` in your project.
```
The installer will now create this file for you and place in the names of the required env variables (but you have to fill in the values). There will also be a randomly generated key for NextAuth. Example:

```
NEXTAUTH_SECRET_KEY=v09ds7f09dsa7g9sadg7a9sg7as96gasg
LIVEBLOCKS_SECRET_KEY=
GITHUB_CLIENT_ID=
GITHUB_SECRET_ID=
```